site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -24,8 +24,6 @@ An object that can hold any information about the Tenant that should be persiste
 [field]#tenant.emailConfiguration.enabled# [type]#[Boolean]# [optional]#Optional# [deprecated]#Deprecated#::
 When this value is set to true the email configuration provided by this tenant will take precedence over the configuration by the System Configuration.
 +
-&nbsp;
-+
 [deprecated]#Removed in version 1.8.0# In version 1.8.0 and beyond, a Tenant's email configuration is enabled upon configuration.
```

Comment:
That is one weird sentence. "a Tenant's email configuration is enabled upon configuration." ?

Reply:
Completely agree.  I think I'm just going to leave it since it applies to version 1.8.0.